libs/ledger-live-common/src/__tests__/test-helpers/bridge.ts

```diff
@@ -627,7 +627,9 @@ export function testBridge<T extends TransactionCommon>(
           });
           makeTest("Default empty amount has an amount error", async () => {
             const account = await getSynced();
-            const t = { ...bridge.createTransaction(account) };
+            const t = await bridge.prepareTransaction(account, {
```

Comment:
For some context, Solana does need a prepare transaction because all the status is generated at this moment.

Reply:
This seems safe, but since it impacts the tests for all coins, can you please run the integ tests again without filtering on solana?

Reply:
Well, none of the _many_ errors seem related to that change... LGTM :upside_down_face: